web/src/user/LibraryPage/ak-library-application-list.ts

```diff
@@ -12,7 +12,7 @@ import PFBase from "@patternfly/patternfly/patternfly-base.css";
 
 import type { Application } from "@goauthentik/api";
 
-import type { AppGroupEntry, AppGroupList } from "./types";
+import type { AppGroupEntry, AppGroupList } from "./types.js";
```

Comment:
should be we importing these files are `.js`? If anything we probably shouldn't for consistency

Reply:
The standard is quickly becoming "If it's going to be a JS file, [always use a .js extension](https://www.typescriptlang.org/docs/handbook/modules/theory.html#module-format-detection)."  I'd like to keep doing that from now on.  Typescript has extensive documentation on it.  Browser stuff should be ES2020 and beyond.